lib/views/after_auth_screens/events/create_event_form.dart

```diff
@@ -61,8 +61,14 @@ class CreateEventForm extends StatelessWidget {
               focusedBorder: InputBorder.none,
               enabledBorder: InputBorder.none,
               suffix: GestureDetector(
-                onTap: () => navigationService.pushScreen(Routes.mapScreen,
-                    arguments: model),
+                onTap: () => navigationService.pushScreen(
+                  Routes.mapScreen,
+                  arguments: {
+                    'model': model,
+                    'latitude': 37.42796133580664,
+                    'longitude': -122.085749655962,
```

Comment:
hey @guptapriyanshu7, i'm going to assume these are placeholder values, can you make a comment above it and just leave a description mentioning these are placeholder values until something else will be in place etc. (something like that)

Reply:
Added description.